linux/LinuxMDEparser/README.ME

```diff
@@ -0,0 +1,23 @@
+# LinuxMDEparser
+Microsoft Defender Endpoint Log Parser for Linux and MacOS.
+
+Compiled sources (with pyinstaller) can be found on dist folder.
+
+# About
+The LinuxMDEparser is a tool to export multiple log files from MDE for Linux and Mac to Excel, making it easier to filter and search the logs.
+
+# Help
```

Comment:
Please give a short step-by-step example on how to use it (from acquiring a log to processing it using LinuxMDEParser, it increases adoption/minimizes "how shall I use it?" kind of questions). I can see that you mentioned where to get files below, what I mean is a short Bash/Zsh transcript

Reply:
I'll update and resubmit README.MD

Reply:
Corrected

Reply:
What I am looking for:

# Compiling tool using [pyinstaller](https://www.pyinstaller.org/)

1) pip install pyinstaller
2) pyinstaller --onefile LinuxMDEparser.py --hidden-import xlsxwriter

pyinstaller produces a standalone executable as dist/LinuxMDEparser (or dist/LinuxMDEparser.exe on Windows)

Reply:
Having build.sh and build.cmd that will do it for user would be even more convenient